src/DeviceRow.vala

```diff
@@ -97,8 +100,25 @@ public class Bluetooth.DeviceRow : Gtk.ListBoxRow {
         settings_button.no_show_all = true;
         settings_button.visible = false;
 
+        forget_button = new Gtk.Button ();
+        forget_button.image = new Gtk.Image.from_icon_name ("edit-delete-symbolic", Gtk.IconSize.MENU);
+        forget_button.get_style_context ().add_class (Gtk.STYLE_CLASS_FLAT);
+        forget_button.tooltip_text = _("Forget this device");
+        forget_button.margin_end = 3;
+        forget_button.show_all ();
+        forget_button.no_show_all = true;
+        forget_button.visible = false;
+        forget_button.button_release_event.connect (() => {
```

Comment:
Can you move this signal connection to the end? This better separates UI creation and logic

Reply:
sure!